client/config/webpackDevServer.config.ts

```diff
@@ -69,7 +69,7 @@ export default function ({
     publicPath,
     // WebpackDevServer is noisy by default so we emit custom message instead
     // by listening to the compiler events with `compiler.plugin` calls above.
-    quiet: true,
+    quiet: false,
```

Comment:
Just curious to know why we are setting `quiet` to `false` here now?

Reply:
just me accidentally leaving this on from debugging watch on node 18, set it back to `true`